advanced-achievements-plugin/src/main/java/com/hm/achievement/config/ConfigurationParser.java

```diff
@@ -199,26 +193,30 @@ private void parseAchievements() throws PluginLoadError {
 				}
 			}
 		}
-
-		int categories = NormalAchievements.values().length + MultipleAchievements.values().length + 1
-				- disabledCategories.size();
-		logger.info("Loaded " + achievementsAndDisplayNames.size() + " achievements in " + categories + " categories.");
-		if (disabledCategories.size() == 1) {
-			logger.info(disabledCategories.size() + " disabled category: " + disabledCategories.toString());
-		} else if (!disabledCategories.isEmpty()) {
-			logger.info(disabledCategories.size() + " disabled categories: " + disabledCategories.toString());
-		}
 	}
 
 	/**
 	 * Parses all achievements for a given category or category + subcategory. Populates the sortedThresholds map.
-	 * 
-	 * @param path
-	 * @throws PluginLoadError
+	 *
+	 * @param path category or category.subcategory
+	 * @throws PluginLoadError If an achievement fails to parse due to misconfiguration.
 	 */
 	private void parseAchievements(String path) throws PluginLoadError {
+		if (disabledCategories.contains(path)) {
+			return;
+		}
+
+		Set<String> keys = mainConfig.getShallowKeys(path);
+
+		// Disable category if no achievements exist
+		// Don't add multi-achievement categories to disabled categories (path has a .)
+		if (keys.isEmpty() && !path.contains(".")) {
+			disabledCategories.add(path);
+			return;
+		}
+
 		List<Long> thresholds = new ArrayList<>();
-		for (String threshold : mainConfig.getShallowKeys(path)) {
+		for (String threshold : keys) {
 			parseAchievement(path + "." + threshold);
 			thresholds.add(Long.valueOf(threshold));
```

Comment:
_Commands_ achievements aren't based on thresholds, so this will blow up as you'll try to be converting a string such as "yourAch1" to a number. 😉

Reply:
Ah I see, reverted that part of changes in 2a00fd4